templates/nextjs/src/app/layout.tsx

```diff
@@ -70,6 +71,9 @@ export default function RootLayout({ children }: RootLayoutProps) {
                   new BakoSafeConnector(),
                   new FueletWalletConnector(),
                   new FuelWalletDevelopmentConnector(),
+                  new SolanaConnector({
+                    fuelProvider: providerToUse,
+                  }),
```

Comment:
https://github.com/FuelLabs/fuel-connectors/blob/e02834af9e3d68acdc1d8223e3cc18af5d889030/packages/connectors/src/defaultConnectors.ts#L38-L40

Is it worthwhile passing a `projectId` variable.

Reply:
I'm going to resolve this, as I don't think there is much importance